analytics.ts

```diff
@@ -0,0 +1,121 @@
+import * as Settings from '@taqueria/protocol/Settings';
+import { chain, map, resolve } from 'fluture';
+import { pipe } from 'https://deno.land/x/fun@v1.0.0/fns.ts';
+import { getMachineId } from 'https://deno.land/x/machine_id@v0.3.0/mod.ts';
+import type { DenoArgs } from './taqueria-types.ts';
+import * as utils from './taqueria-utils/taqueria-utils.ts';
+
+const {
+	doesPathExist,
+	readJsonFile,
+	writeJsonFile,
+	eager,
+} = utils.inject({
+	stdout: Deno.stdout,
+	stderr: Deno.stderr,
+});
+
+const consentPrompt =
+	'Allow Taqueria anonymously report usage statistics, in accordance with the privacy policy to help make Taqueria better? [y/yes] or [n/no]';
```

Comment:
Let's make this:

"Help improve Taqueria by sharing anonymous usage statistics in accordance with the privacy policy?"